README.md

```diff
@@ -1,132 +1,127 @@
+## Usage Example
 
-* Materials and Geometry are combined in world objects.
-* Decoupled cameras and controllers.
-* The code for the render engines is decoupled from the objects, allowing multiple render engines (e.g. wgpu and svg).
+> **Note**
+> A walkthrough of this example can be found in [the
+> guide](https://pygfx.readthedocs.io/en/latest/guide.html#how-to-use-pygfx).
 
-Further we aim for a few niceties:
-* Proper support for high-res screens.
-* Builtin anti-aliasing.
-* Custom post-processing steps.
-* Support for picking objects and parts within objects.
-* (approximate) order-independent transparency (OIT) (not implemented yet).
+```python
+import pygfx as gfx
 
+cube = gfx.Mesh(
+    gfx.box_geometry(200, 200, 200),
+    gfx.MeshPhongMaterial(color="#336699"),
+)
 
-## WGPU is awesome (but also very new)
+def animate():
+    rot = gfx.linalg.Quaternion().set_from_euler(
+            gfx.linalg.Euler(0, 0.01)
+        )
+    cube.rotation.multiply(rot)
 
-Working with the WGPU API feels so much nicer than OpenGL. It's well
-defined, no global state, we can use compute shaders, use storage
-buffers (random access), etc.
+if __name__ == "__main__":
+    gfx.show(cube, before_render=animate)
 
-Fair enough, the WGPU API is very new and is still changing a lot, but
-eventually it will become stable. One of the biggest downsides right
-now is the lack of software rendering. No luck trying to run wgpu on a
-VM or CI.
+```
+<img src="./docs/_static/guide_rotating_cube.gif" alt="drawing" width="400"/>
 
-Because of how Vulkan et. al. work, the WGPU API is aimed at predefining
-objects and pipelines and then executing these. Almost everything is
-"prepared". The main reasoning for this is consistency and stable drivers,
-but it also has a big advantage for us Pythoneers: the amount of code per-draw-per-object
-is very limited. This means we can have *a lot* of objects and still be fast.
 
-As an example, see `collections_line.py`: drawing 1000 line objects with 30k points each at 57 FPS (on my laptop).
+## Feature Highlights
+Some of pygfx's key features are:
 
+- SDF based rendering (example)
+- order-independent transparency (OIT) (example)
+- a shader templating system (example)
+- event system with built-in picking (example)
+- Texture and color mapping supporting 1D, 2D and **3D data** (example)
 
-## How to build a visialization
 
-See also the examples, they all do something like this:
+And many more! Check out our [feature
+demos](https://pygfx.readthedocs.io/en/latest/_gallery/index.html) in the docs.
 
-* Instantiate a renderer and a canvas to render to.
-* Create a scene and populate it with world objects.
-* Create a camera (and maybe a control).
-* Define an  animate function that calls: `renderer.render(scene, camera)`
+## About pygfx
 
+Pygfx is a graphics library that uses WGPU (the successor of OpenGL) to provide
+GPU acceleration to rendering workloads. It mimics the ThreeJS API and is geared
```

Comment:
It's more "inspired by" than "mimics", because we (deliberately) in some cases :)